src/Core/src/Platform/ActivationState.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+namespace Microsoft.Maui
+{
+	public class ActivationState : IActivationState
```

Comment:
Merged all the files into one because it is simpler. I did this because I was going to make `IActivationState` only use `IMauiWindowContext`, but then realised that for compatibility we also need to make a context for `Forms.Init()` when the application does not yet have an activity.